docs/make.jl

```diff
@@ -13,6 +13,9 @@ makedocs(;
     ),
     pages=[
         "Home" => "index.md",
+        "User guide" => [
+            "Rules" => "manual/rules.md",
+        ]
```

Comment:
```suggestion
        ],
```
trailing comma is always useful to reduce diff later when adding new lines.

Reply:
Done.